bulldozer/signals_test.go

```diff
@@ -31,7 +31,7 @@ func TestSignalsMatches(t *testing.T) {
 		Comments:          []string{"FULL_COMMENT_PLZ_MERGE"},
 		CommentSubstrings: []string{":+1:"},
 		PRBodySubstrings:  []string{"BODY_MERGE_PLZ"},
-		Branches:          []string{"develop"},
+		Branches:          []string{"develop", "test/*"},
```

Comment:
Updated to use regular expression syntax, though it is strange that `regexp.MatchString(a, b)` also accepts glob patterns - the initial version of this PR is what I had been running internally in our team.

(Golang is new to me, so this "dual" behaviour seems odd)

Reply:
`regexp.MatchString` does not accept glob patterns, but every glob pattern is a valid regular expression (though with a different meaning.) Because it finds a match at any position (and does not require a complete match), using `test/*` (the string `test` followed by 0 or more `/` characters) matches the expected name but also matches other things:

- `test/my-branch`
- `test//////my-branch`
- `testing`
- `not-a-test`